### ISSUE TITLE
task grounding-dino-task output keys are missing

### ISSUE THREAD

**Issue by GS-Albert** [X-PLUG/MobileAgent#113]
你好，我尝试了很多次，时不时会出现这种情况：

### Perceptor ... ###

2025-03-07 16:39:45,211 - modelscope - WARNING - task grounding-dino-task input definition is missing
FutureWarning: The `device` argument is deprecated and will be removed in v5 of Transformers.
UserWarning: torch.utils.checkpoint: the use_reentrant parameter should be passed explicitly. In version 2.5 we will raise an exception if use_reentrant is not passed. use_reentrant=False is recommended, but if you need to preserve the current default behavior, you can pass use_reentrant=True. Refer to docs for more details on the differences between the two variants.
UserWarning: None of the inputs have requires_grad=True. Gradients will be None
FutureWarning: `torch.cuda.amp.autocast(args...)` is deprecated. Please use `torch.amp.autocast('cuda', args...)` instead.
SupervisionWarnings: annotate is deprecated: `BoxAnnotator` is deprecated and will be removed in `supervision-0.22.0`. Use `BoundingBoxAnnotator` and `LabelAnnotator` instead
2025-03-07 16:39:49,148 - modelscope - WARNING - task grounding-dino-task output keys are missing

页面一直卡在这里，没有变化。请问是什么原因？

**Comment by junyangwang0410** [comment]
你好，如果GroundingDINO的运行速度很慢，可能是使用了CPU，可以尝试使用带有GPU的机器运行
